tests/integration/test_chat.py

```diff
@@ -916,6 +927,28 @@ def test_restart_button(self):
         bot_messages = self.element.find_elements_by_css_selector('.messages .message.bot')
         self.assertEqual(len(bot_messages), 1)
 
+    @data("", "This is a test subject")
+    def test_chat_subject(self, chat_subject):
+        self.configure_block(yaml_good, subject=chat_subject)
+        self.element = self.go_to_view("student_view")
+        subject_element = self.element.find_elements_by_css_selector('div.subject')
+
+        if chat_subject:
+            self.assertEqual(len(subject_element), 1)
```

Comment:
Added the check. I used `self.assertTrue(chat_subject in subject_element[0].text)`, to make the test more resilient to any changes in the way we display the contents.